pkg/policies/binary/binary.go

```diff
@@ -172,23 +173,43 @@ func (b Binary) Check(ctx context.Context, c *github.Client, owner,
 		return nil, res.Error2
 	}
 
+	logs := convertLogs(l.logs)
+	pass := res.Score >= checker.MaxResultScore
 	var notify string
-	if res.Score < checker.MaxResultScore {
-		notify = fmt.Sprintf("Scorecard Check Binary Artifacts failed: %v\n"+
-			"Please run scorecard directly for details: https://github.com/ossf/scorecard\n",
+	if !pass {
+		notify = fmt.Sprintf(
+			"Scorecard Check Binary Artifacts failed: %v\n"+
+				"Binary Artifacts are an increased security risk in your repository. Binary artifacts cannot be reviewed, allowing possible obsolete or maliciously subverted executables.\n"+
```

Comment:
minor nit: "allowing THE INTRODUCTION OF POSSIBLY obsolete or maliciously..."

Reply:
Fixed